cli/src/wallet.rs

```diff
@@ -895,7 +895,9 @@ fn process_show_stake_account(
             Ok("".to_string())
         }
         Ok(StakeState::RewardsPool) => Ok("Stake account is a rewards pool".to_string()),
-        Ok(StakeState::Uninitialized) => Ok("Stake account is uninitialized".to_string()),
+        Ok(StakeState::Uninitialized) | Ok(StakeState::Lockup(_)) => {
+            Ok("Stake account is uninitialized".to_string())
```

Comment:
probably want a different error for each branch just for debugging

Reply:
I have an issue specifically for errors in stake_api